float64[6] ee_vel
float64[6] force
float64[6] force_des